test/server/cards/01_SOR/units/SteadfastBattalion.spec.js

```diff
@@ -0,0 +1,50 @@
+describe('Steadfast Battalion', function() {
+    integration(function() {
+        describe('Steadfast Battalion\'s ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        base: 'security-complex',
+                        groundArena: ['steadfast-battalion', 'battlefield-marine'],
+                        leader: { card: 'ig88#ruthless-bounty-hunter', deployed: true }
+                    },
+                    player2: {
+                        groundArena: ['frontier-atrt', 'wampa'],
+                        leader: { card: 'boba-fett#daimyo', deployed: true }
+                    }
+                });
+            });
+
+            it('should give a unit +2/+2', function () {
```

Comment:
needs condition text: "should give a unit +2/+2 if the player controls a leader". Ditto for other test